recipe/meta.yaml

```
@@ -11,18 +11,19 @@ source:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
+  skip: true  # [py<36]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
-    - python >=3.4
+    - python
     - pip
-    - pytest-runner
+    - setuptools
+    - wheel
   run:
-    - python >=3.4
+    - python
     - cycler >=0.10.0
-    - matplotlib-base >=2.0.2,!=3.0.0
+    - matplotlib >=2.0.2,!=3.0.0
```

```suggestion
    - matplotlib-base >=2.0.2,!=3.0.0
```
matplotlib-base is fine because matplotlib has pyqt and it will increase the package size significantly